crates/sui-types/src/sui_system_state/sui_system_state_inner_v1.rs

```diff
@@ -28,6 +29,7 @@ const E_METADATA_INVALID_NET_ADDR: u64 = 4;
 const E_METADATA_INVALID_P2P_ADDR: u64 = 5;
 const E_METADATA_INVALID_PRIMARY_ADDR: u64 = 6;
 const E_METADATA_INVALID_WORKER_ADDR: u64 = 7;
+const E_METADATA_INVALID_POP: u64 = 9;
```

Comment:
why 9 and not 8?

Reply:
https://github.com/MystenLabs/sui/blob/4189171efdca356f0e12b043478f895ed169765b/crates/sui-framework/sources/governance/validator.move there are new error code added to move (that is thrown from move code and no rust code reference). switched orders to make rust error codes are always non gapped.